dev.zed.Zed.Preview.yaml

```diff
@@ -0,0 +1,210 @@
+      - /share/gtk-doc
+      - /share/info
+      - /share/man
+      - /share/pkgconfig
+      - "*.a"
+      - "*.la"
+    config-opts:
+      - -Dgtk_doc=false
+      - -Dintrospection=false
+    sources:
+      - type: archive
+        sha256: bad10f3c553a0e1854649ab59c5b2434da22ca1a54ae6138f1f53961567e1ab7
+        url: https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
+
+  - name: libmd
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://libbsd.freedesktop.org/releases/libmd-1.1.0.tar.xz
+        sha256: 1bd6aa42275313af3141c7cf2e5b964e8b1fd488025caf2f971f43b00776b332
+        x-checker-data:
+          type: anitya
+          project-id: 15525
+          stable-only: true
+          url-template: https://libbsd.freedesktop.org/releases/libmd-$version.tar.xz
+    cleanup:
+      - /lib/pkgconfig
+      - /share/man
+
+  - name: libbsd
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://libbsd.freedesktop.org/releases/libbsd-0.12.2.tar.xz
+        sha256: b88cc9163d0c652aaf39a99991d974ddba1c3a9711db8f1b5838af2a14731014
+        x-checker-data:
+          type: anitya
+          project-id: 1567
+          stable-only: true
+          url-template: https://libbsd.freedesktop.org/releases/libbsd-$version.tar.xz
+    cleanup:
+      - /lib/pkgconfig
+      - /share/man
+
+  - name: netcat
+    buildsystem: simple
+    build-commands:
+      - |
+        while read patch; do
+          echo "Applying $patch..."
+          patch -Np1 -i "debian/patches/$patch"
+        done < debian/patches/series
+      - make CFLAGS="$CFLAGS -I/usr/include/libbsd" LDFLAGS="$LDFLAGS -lbsd"
+      - install -Dm0755 nc /app/bin/nc
+    sources:
+      - type: archive
+        url: https://salsa.debian.org/debian/netcat-openbsd/-/archive/debian/1.228-1/netcat-openbsd-debian-1.228-1.tar.gz
+        sha256: a1eb2593335123efc6e9c12a7e73c72cbf0b665739130fc383322f36757f37cc
+        x-checker-data:
+          type: anitya
+          project-id: 21535
+          stable-only: true
+          url-template: https://salsa.debian.org/debian/netcat-openbsd/-/archive/debian/$version-1/netcat-openbsd-debian-$version-1.tar.gz
+
+  - name: zed
+    buildsystem: simple
+    build-commands:
+      - install -Dm 755 bin/* --target-directory ${FLATPAK_DEST}/bin
+      - install -Dm 755 lib/* --target-directory ${FLATPAK_DEST}/lib
+      - install -Dm 755 libexec/* --target-directory ${FLATPAK_DEST}/libexec
+      - install -Dm 644 share/applications/* --target-directory ${FLATPAK_DEST}/share/applications
+      - install -Dm 644 ${FLATPAK_ID}.metainfo.xml --target-directory ${FLATPAK_DEST}/share/metainfo
+      - install -Dm 644 share/icons/hicolor/512x512/apps/zed.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/zed-preview.png
+
+      # Rename instances of `zed` to `${FLATPAK_ID}`
+      - rename zed-preview ${FLATPAK_ID} ${FLATPAK_DEST}/share/{applications/*,icons/hicolor/*/apps/*}
```

Comment:
`rename-icon`, `rename-desktop-file` in the manifest exist for this kind of things.

Reply:
Nice, I didn't know about them

Reply:
it's in the doc.

Reply:
wait, actually those won't be useful. Here we are renaming `zed-preview` in those files to `$FLATPAK_ID`, not renaming the files themselves.